config.w32

```diff
@@ -235,17 +235,17 @@ if (PHP_MONGODB != "no") {
   if (PHP_MONGODB_SASL != "no" &&
       CHECK_LIB("libsasl.lib", "mongodb", PHP_MONGODB) &&
       CHECK_HEADER_ADD_INCLUDE("sasl/sasl.h", "CFLAGS_MONGODB")) {
```

Comment:
Suggest removing checks for the Cyrus `libsasl.lib` and `sasl/sasl.h`:

```sh
if (PHP_MONGODB_SASL != "no") {
```

The C driver does not appear to have header/library checks when configuring with `ENABLE_SASL=SSPI`. I expect SSPI is assumed to be present on Windows.

Reply:
Thank you. I wasn't sure whether those were still needed, so I erred on the side of caution, but removed the checks now.